src/rpdk/core/data/schema/provider.definition.schema.v1.json

```diff
@@ -274,65 +87,53 @@
                 "AWS::EC2::Instance",
                 "Initech::TPS::Report"
             ],
-            "type": "string",
-            "pattern": "^[a-zA-Z0-9]{2,64}::[a-zA-Z0-9]{2,64}::[a-zA-Z0-9]{2,64}$"
+            "$ref": "base.definition.schema.v1.json#/properties/typeName"
         },
         "$comment": {
-            "$ref": "http://json-schema.org/draft-07/schema#/properties/$comment"
+            "$ref": "base.definition.schema.v1.json#/properties/$comment"
         },
         "title": {
-            "$ref": "http://json-schema.org/draft-07/schema#/properties/title"
+            "$ref": "base.definition.schema.v1.json#/properties/title"
         },
         "description": {
             "$comment": "A short description of the resource provider. This will be shown in the AWS CloudFormation console.",
-            "$ref": "http://json-schema.org/draft-07/schema#/properties/description"
+            "$ref": "base.definition.schema.v1.json#/properties/description"
         },
```

Comment:
our definitions are still the same anyways, right? why switch these to our own definitions?